core/plugins/org.polarsys.capella.core.data.migration/src/org/polarsys/capella/core/data/migration/cmdline/MigrationCommandLine.java

```diff
@@ -104,15 +109,29 @@ public void migrateAllImportedProjects(Shell shell) {
    *
    */
   public void setRefreshPrefs() {
+      IPreferenceStore preferenceStore = SiriusEditPlugin.getPlugin().getPreferenceStore();
+      initialValue_RefreshOnOpening = preferenceStore.getBoolean(SiriusUIPreferencesKeys.PREF_REFRESH_ON_REPRESENTATION_OPENING.name());
```

Comment:
Is there a differences between accessing the preferences by using the `preferenceStore` like we here, compared to `InstanceScope.INSTANCE.getNode(SiriusEditPlugin.ID);` ? 

Because for obtaining the preferences we do it in the first way, and for resetting them we do it another way. I suppose the result is the same, but this is kinda confusing.